packages/modules-runtime/server.js

```diff
@@ -15,7 +15,7 @@ makeInstallerOptions.fallback = function (id, parentId, error) {
   if (topLevelIdPattern.test(id)) {
     if (typeof Npm === "object" &&
         typeof Npm.require === "function") {
-      return Npm.require(id);
+      return Npm.require(id, error);
```

Comment:
Intentionally undocumented API: you can now pass an error object as the second argument to `Npm.require` to control what gets thrown if the import fails.